plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayAccountConfigExtension.kt

```diff
@@ -1,12 +1,20 @@
 package com.github.triplet.gradle.play
 
 import com.github.triplet.gradle.play.internal.AccountConfig
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
 import java.io.File
 
 data class PlayAccountConfigExtension @JvmOverloads constructor(
-        internal val name: String = "", // Needed for Gradle
+        @get:Internal internal val name: String = "", // Needed for Gradle
```

Comment:
The comment can be shifted as `value` into the annotation:
```
@get:Internal("Needed for Gradle")
```

Not sure what we should prefer, though…

Reply:
Let's leave it as is because we can't remove the property at all. If it were just a question of visibility, then we would move the comment to the annotation.

Reply:
SGTM